src/init.cpp

```diff
@@ -67,6 +68,9 @@
 #include <signal.h>
 #endif
 
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/split.hpp>
+
```

Comment:
nit: no need for a blank line here, could just incorporate these 2 new includes to the ones below.